scripts/get_golang_version.sh

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
-# Required GOLANG Version for project specified here
-GOLANG_VERSION="1.14.7"
 
-# Check to make sure that it matches what is specified in go.mod
+# Required GOLANG versions for project specified here.
```

Comment:
A little more doc here would be helpful to explain the reason for the 'all', 'dev', and default outputs.

Reply:
Will add comments.